stable/osg-hosted-ce/osg-hosted-ce/templates/deployment.yaml

```diff
@@ -23,9 +23,14 @@ spec:
         release: {{ .Release.Name }}
         instance: {{ .Values.Instance }}
     spec:
-      {{ if .Values.NodeSelection }}
+      {{ if .Values.NodeSelection.Enabled }}
       nodeSelector: 
-        kubernetes.io/hostname: {{ .Values.NodeSelection }}
+        {{ if .Values.NodeSelection.Hostname }}
+        kubernetes.io/hostname: {{ .Values.NodeSelection.Hostname }}
+        {{ end }}
```

Comment:
Wouldn't this result in two hostname settings if the NodeSelection.Hostname is set?

Reply:
I'm a little confused, isn't .Values.NodeSelection.Hostname just a variable from values? Or does yaml not work that way?

Reply:
nm, I misread the patch and assumed the - line was still in the file.  This is fine.  Sorry about the noise here.